code/datums/components/uplink.dm

```diff
@@ -147,6 +147,15 @@ GLOBAL_LIST_EMPTY(uplinks)
 							is_inaccessible = 0
 					if(is_inaccessible)
 						continue
+				if(I.restricted_species.len)
+					var/is_inaccessible = TRUE
+					var/mob/living/carbon/human/H = user
+					for(var/F in I.restricted_species)
+						if(F == H.dna.species.id)
```

Comment:
Does this cause problems when the user is nonhuman (syndrone)?

Reply:
The user isn't guaranteed to be human. You need a check for that

Reply:
Would've fixed this when swindle brought this up but hurr durr people don't need tab spaces when using a phone

Reply:
Human check added.